### DISCUSSION TITLE
SRT Ingest Probe Failure

### DISCUSSION THREAD

**Discussion by JeffThis** [datarhei/restreamer#672] [Q&A]
Hi, I recently ran across this project and decided to give it a try. After installing and ensuring the correct ports are accounted for, I followed the "how to stream SRT from OBS" tutorial video to get started.

After copying the SRT ingest URL, the encoder shows that it is connected successfully to the server, however when I click next in the Restreamer wizard, it attempts to probe the stream and then shows "failed to verify the source."

The logs do show that it received the SRT stream, but has the error "no publisher for this resource found." I'm sure I'm missing something obvious, but can someone please help me understand what this error indicates?

ts=2024-01-07T01:50:35Z level=INFO component="Session" msg="Active" id="c761fb1a-cd22-45c7-8e8f-594416f5d81c" location="publish:c761fb1a-cd22-45c7-8e8f-594416f5d81c" peer="xx.xxx.xxx.xxx:63750" reference="c761fb1a-cd22-45c7-8e8f-594416f5d81c" type="srt"
ts=2024-01-07T01:50:35Z level=INFO component="SRT" address=":6000" client="xxx.xxx.xxx.xxx:63750" handler="PUBLISH" resource="c761fb1a-cd22-45c7-8e8f-594416f5d81c" status="START"
ts=2024-01-07T01:51:02Z level=INFO component="Process" msg="Created" id="restreamer-ui:ingest:c761fb1a-cd22-45c7-8e8f-594416f5d81c_probe"
ts=2024-01-07T01:51:02Z level=INFO component="Process" msg="Created" id="restreamer-ui:ingest:c761fb1a-cd22-45c7-8e8f-594416f5d81c_probe"
ts=2024-01-07T01:51:02Z level=INFO component="Process" msg="Starting" id="restreamer-ui:ingest:c761fb1a-cd22-45c7-8e8f-594416f5d81c_probe"
ts=2024-01-07T01:51:02Z level=INFO component="Process" msg="Started" id="restreamer-ui:ingest:c761fb1a-cd22-45c7-8e8f-594416f5d81c_probe"
ts=2024-01-07T01:51:02Z level=INFO component="SRT" msg="no publisher for this resource found" address=":6000" client="127.0.0.1:57958" handler="CONNECT" resource="c761fb1a-cd22-45c7-8e8f-594416f5d81c.stream" status="NOTFOUND"
ts=2024-01-07T01:51:05Z level=INFO component="Process" msg="Failed" id="restreamer-ui:ingest:c761fb1a-cd22-45c7-8e8f-594416f5d81c_probe"
ts=2024-01-07T01:51:05Z level=INFO component="Process" msg="Stopped" id="restreamer-ui:ingest:c761fb1a-cd22-45c7-8e8f-594416f5d81c_probe"

To be sure there is nothing wrong with my SRT source, I tested with an FFMPEG SRT listener on the same server that is hosting the Restreamer container and it was able to receive the stream successfully.

**Comment by JeffThis** [comment]
I had assumed I must be missing a step since this is my first time trying this project, but it appears to actually be a bug, this issue: https://github.com/datarhei/restreamer/issues/670

**Comment by ioppermann** [comment]
There's currently a bug in the UI that displays a wrong SRT URL to be pasted into OBS. See this description: https://github.com/datarhei/restreamer/issues/635#issuecomment-1837234576

This bug has been fixed and the fix will be available in the next release.
